src/main/kotlin/org/rust/ide/refactoring/extractFunction/RsExtractFunctionConfig.kt

```diff
@@ -127,7 +128,10 @@ class RsExtractFunctionConfig private constructor(
             val body = if (single is RsBlockExpr) {
                 single.block.text
             } else {
-                val stmts = elements.map { it.text }.toMutableList()
+                val stmts = parameters.filter { !it.isSelected }.map { "let ${it.name}: ${it.type} = ;" }.toMutableList()
```

Comment:
Maybe `${it.type}::default()` is better?

Reply:
IMO the type should not be inserted explicitly. And also check `InitializeWithDefaultValueFix`, maybe you can reuse it somehow.

Reply:
Sorry for so long.

I don't seem to find a more convenient way to reuse the contents of `InitializeWithDefaultValueFix`, maybe there are other ways?

Reply:
OK, let's just use `let ${it.name}: ${it.type};`, without `=`. Then a user can apply `InitializeWithDefaultValueFix` when the caret is located under any usage of the variable.